src/openms/source/KERNEL/MSExperiment.cpp

```diff
@@ -284,13 +284,6 @@ namespace OpenMS
       // update range of EACH chrom, if we need them individually later
       cp.updateRanges();
 
-      // ignore TICs and ECs for the whole experiments range (as these are usually positioned at 0 and therefor lead to a large white margin in plots if included)
```

Comment:
as discussed in previous issue, chromatograms are now not excluded in range calculations.
This is subject to further discussions...

Reply:
TOPPView will display an empty area in 2D now (if chroms are not taken care of in TOPPView separately; don't recall at the moment).